about/package-scope.md

```diff
@@ -18,21 +18,257 @@ Currently, the packages that pyOpenSci reviews also need to fall into the
 technical and applied scope of our organization. This scope may expand over time
 as the organization grows.
 
-
-
 ## Is Your Package in Scope For pyOpenSci Review?
 
-pyOpenSci only reviews packages that fall within our specified domain and
-technical scope listed below.
+pyOpenSci reviews packages that fall within a list of specified categories and
+domains. Packages must also meet our technical scope requirements.
 
 If you are unsure whether your package is in scope for review, please
 open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSci/software-review/issues/new?assignees=&labels=0%2Fpresubmission&template=presubmission-inquiry.md&title=) to get feedback from
 one of our editors. We are happy to look at your package and help you understand
 whether it is in scope or not.
 
-```{include} /appendices/scope.md
+### About the types of packages that we review
+
+pyOpenSci reviews packages that support open reproducible science,
+data processing and and the various stages of managing the
+data lifecycle. Packages submitted to pyOpenSci should fit into one or
+more of the categories below and should be within our technical scope.
+
+```{admonition} Package Use Metrics Are Not a Requirement for Review
```

Comment:
This suggests "we don't need you to go and measure your package use (but we may consider how much it is used)".  Rather: "Widespread package usage is not required for review", "We review independent of popularity", or similar?

Reply:
many thanks @stefanv fixing. @NickleDave i think had similar feedback here and i haven't been able to get it right!

Reply:
is this any better :

Your Package Does Not Need to Have Widespread Use to be Reviewed
:class: important

We review packages across a spectrum of small to large user bases. The popularity of your package is not a consideration in our review process!

@NickleDave

Reply:
Sounds good to me!

nit: would maybe say it like "Your Package Does Not Need to Be In Widespread Use Yet to be Reviewed" or "Your Package Does Not Need to Be Widely Used to be Reviewed"? ("in use" not "have use")

Reply:
this was merged a few hours ago... would you be open to creating a PR for this - i'm fine with cleaner wording.

Reply:
Yup, sorry, was catching up with replies